employee-rostering-frontend/src/ui/pages/shift/ReactBigCalendarOverrides.css

```diff
@@ -32,22 +35,152 @@
 }
 /* End of Hack */
 
+.rbc-header.published-day::before {
+    content: " ";
+    font-family: "pficon";
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-style: normal;
+    font-weight: normal;
+    font-feature-settings: normal;
+    font-variant: normal;
+    text-decoration: none;
+    text-transform: none;
+}
+
+.rbc-header.draft-day::before {
+    content: " ";
+    font-family: "pficon";
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-style: normal;
+    font-weight: normal;
+    font-feature-settings: normal;
+    font-variant: normal;
+    text-decoration: none;
+    text-transform: none;
+}
+
+.availability-wrapper {
+    position: absolute !important;
+    top: 0 !important;
+    left: 0 !important;
+    width: 100% !important;
+    height: 100% !important;
+    opacity: 0.75;
+}
+
+.availability-event {
+    display: "flex";
```

Comment:
@Christopher-Chianelli This is not a valid value for the `display` property. If you want to use `flex` use `display: flex` (https://developer.mozilla.org/en-US/docs/Web/CSS/flex#Example).